primitives/npos-elections/src/lib.rs

```diff
@@ -117,9 +119,6 @@ pub use sp_arithmetic;
 #[doc(hidden)]
 pub use sp_std;
 
-// re-export the solution type macro.
-pub use sp_npos_elections_solution_type::generate_solution_type;
```

Comment:
yeah this is what I mean, since the crate names are already long, I think we should re-export `frame-election-provider-solution-type` from `frame-election-provider-support`, which is more commonly imported.